public/data/FidalMathew.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "Fidal Mathew",
+    "bio": "Frontend Developer | I love exploring and learning new technologies. ",
+    "avatar": "https://github.com/FidalMathew.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/FidalMathew",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/fidalmathew10",
+        "icon": "twitter"
+      }
+    ]
+    
+  }
```

```suggestion
    ]  
  }
```
Please Remove that extra line